### PR TITLE
HackerRank Challenges 26-08

### PR DESCRIPTION
**Pull request by Williiam8**

### CODE REVIEW

File: WilliamsGarcia/26-08/HackerRank/InheritanceTwo/InheritanceTwo.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.company;
```
Comment (luisod78) [inline WilliamsGarcia/26-08/HackerRank/InheritanceTwo/InheritanceTwo.java:1]: Maybe you can re-arrange your classes into packages names like excercises.sampleone or bootcamp.dayone or so
What do you think??

**Review by luisod78 — CHANGES_REQUESTED** [review]

File: WilliamsGarcia/26-08/HackerRank/InheritanceTwo/InheritanceTwo.java (hunk 1 above)
Comment (Williiam8) [reply WilliamsGarcia/26-08/HackerRank/InheritanceTwo/InheritanceTwo.java:1]: Sure, I'll work it. Ty

**Review by luisalanmz — APPROVED** [review]
Ok

**Review by edbermeo — APPROVED** [review]
LGTM

**Review by luisod78 — APPROVED** [review]

**Pull request merged**